src/pages/faq.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: "../layouts/Page.astro"
+title: "FAQ"
+---
+
+
+# FAQ
+
+- **What's the difference between MMR and RP?**
+
+    ### MMR
+    Each player currently has a matchmaking rating (MMR) representing their skill and a rating uncertainty that is updated after every game they play. The whole matchmaking system and ladder is built around this. Currently, the default MMR is 1500, and the best players have reached 2300 MMR on the closed beta.
```

Comment:
I would refrain from this highest reached because then we may need to update this doc often 😅

Reply:
Fair point, removed :P